src/parse/parse.service.ts

```diff
@@ -14,6 +14,15 @@ export class ParseService {
         }
     }
 
+    async fetchSerials(url = 'https://uaserial.club/serial/'): Promise<string> {
```

Comment:
we can use fetchContent. Don't need to create one more function

Reply:
+